src/phylum/constants.py

```diff
@@ -43,6 +43,7 @@
     # C#
     "*.csproj": "msbuild",
     "packages.lock.json": "nuget",
+    "packages.*.lock.json": "nuget",
```

Comment:
If it can parse a `packages.lock.json`, it can also parse a `packages.*.lock.json`, it just doesn't do automatic detection.

So I don't think this is a breaking change?

Reply:
Confirmed with local testing. Change coming...